pkgs/by-name/ra/radarr/package.nix

```diff
@@ -0,0 +1,205 @@
+{
+  lib,
+  stdenvNoCC,
+  fetchFromGitHub,
+  buildDotnetModule,
+  dotnetCorePackages,
+  sqlite,
+  withFFmpeg ? true, # replace bundled ffprobe binary with symlink to ffmpeg package.
+  ffmpeg,
+  fetchYarnDeps,
+  yarn,
+  fixup-yarn-lock,
+  nodejs,
+  nixosTests,
+  # update script
+  writers,
+  python3Packages,
+  nix,
+  prefetch-yarn-deps,
+  fetchpatch,
+  applyPatches,
+}:
+let
+  version = "5.19.2.9720";
+  # The dotnet8 compatibility patches also change `yarn.lock`, so we must pass
+  # the already patched lockfile to `fetchYarnDeps`.
+  src = applyPatches {
+    src = fetchFromGitHub {
+      owner = "Radarr";
+      repo = "Radarr";
+      tag = "v${version}";
+      hash = "sha256-9uOh31ezK6+jumGZDco4OzyNvGRP4shlI7KcvoqXizQ=";
+    };
+    patches =
+      [
+        ./nuget-config.patch
+      ]
+      ++ lib.optionals (lib.versionOlder version "6.0") [
+        # See https://github.com/Sonarr/Sonarr/issues/7442 and
+        # https://github.com/Sonarr/Sonarr/pull/7443.
+        # Unfortunately, the .NET 8 upgrade was only merged into the v5 branch,
+        # and it may take some time for that to become stable.
+        # However, the patches cleanly apply to v4 as well.
```

Comment:
```suggestion
        # See https://github.com/Radarr/Radarr/pull/10258
        # Unfortunately, the .NET 8 upgrade will be merged into the v6 branch,
        # and it may take some time for that to become stable.
        # However, the patches cleanly apply to v5 as well.
```